OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs

```diff
@@ -32,5 +33,17 @@ public Actor[] GetGroundAttackers()
 				.Where(a => a.Owner == Player && !a.IsDead && a.IsInWorld && a.HasTrait<Mobile>())
 				.ToArray();
 		}
+
+		[Desc("Gives the player money. Don't use to reduce.")]
+		public void GiveMoney(int amount)
+		{
+			Player.PlayerActor.Trait<PlayerResources>().GiveCash(amount);
+		}
+
+		[Desc("Takes money from the player.")]
+		public void TakeMoney(int amount)
+		{
+			Player.PlayerActor.Trait<PlayerResources>().TakeCash(amount);
```

Comment:
ResourceProperties already has a `Cash` property for adding/removing cash, so these are entirely unnecessary.